app/models/cookbook.rb

```diff
@@ -138,16 +139,21 @@ def sorted_cookbook_versions
   #
   def transfer_ownership(initiator, recipient)
     if initiator.is?(:admin) || collaborator_users.include?(recipient)
-      collaborator = Collaborator.new(user_id: user_id, resourceable: self)
       update_attribute(:user_id, recipient.id)
-      collaborator.save!
+
+      unless collaborators.where(user_id: initiator.id).present?
```

Comment:
Handle the unlikely but possible scenario when a group may have a member who is NOT a collaborator on a resource the group is associated with.  This is also prevented by logic on the controller...but I want to ensure that we have error handling in case the scenario is caused by a rails console change or something to that effect.